MapboxVision/src/main/java/com/mapbox/vision/sync/telemetry/TotalBytesCounter.kt

```diff
@@ -15,45 +17,51 @@ internal interface TotalBytesCounter {
 
     fun millisToNextSession(): Long
 
+    fun fitInLimitCurrentSession(bytes: Long): Boolean
+
     fun fitInLimit(bytes: Long): Boolean
 
     class Impl(
         private val sessionLengthMillis: Long = SESSION_LENGTH_MILLIS,
         private val sessionMaxBytes: Long = SESSION_MAX_BYTES,
-        private val time: Time = Time.SystemImpl
+        private val currentTime: Time = Time.SystemImpl,
+        private val totalBytesCounterPrefs: TotalBytesCounterPrefs = TotalBytesCounterPrefs.Impl("")
```

Comment:
No Need to send default value for `totalBytesCounterPrefs `

Reply:
done